packages/@sanity/portable-text-editor/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "@sanity/portable-text-editor",
+  "version": "2.14.0",
+  "description": "Portable Text Editor made in React",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "author": "Sanity.io <hello@sanity.io>",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "ttsc --module commonjs",
+    "start": "cd demo && vite .",
+    "clean": "rimraf lib coverage",
+    "posttest": "tsc --noEmit --emitDeclarationOnly false",
+    "test:watch": "jest --watch",
+    "test": "jest",
+    "watch": "ttsc --module commonjs --watch",
+    "gen-doc": "typedoc --out docs src",
+    "prettier": "prettier --write './**/*.{ts,tsx,js,css,html}'",
+    "eslint": "eslint 'src/**/*.{ts,js,tsx}'"
+  },
+  "files": [
+    "lib/**/*"
```

Comment:
I think `dist/**/*` needs to be included here as well

Reply:
Looks like no other packages has this property. Removing it all together.